tests/unit/commands/samconfig/test_samconfig.py

```diff
@@ -122,7 +122,7 @@ def test_build(self, do_cli_mock):
 
             LOG.debug(Path(config_path).read_text())
             runner = CliRunner()
-            result = runner.invoke(cli, [])
+            result = runner.invoke(cli, ["--use-container"])
```

Comment:
Why are we adding `--use-container` to all of these? It's unclear why we need to update this.

Reply:
Because otherwise as the CliRunner doesn't see this tag, it fails the invocation and showcases the ContainerTag check error

Reply:
I might be missing something but my initial feeling is that seems wrong. Wouldn't this mean we are requiring `--use-container` now? The command shouldn't fail if we don't provided `--use-container` to the cli.

Reply:
It should now, we are going to fail any use of container related tags if we don't provide --use-container to the cli instead of ignoring them now